hamilton/registry.py

```diff
@@ -79,6 +79,9 @@ def load_extension(plugin_module: ExtensionName):
         assert hasattr(
             mod, f"fill_with_scalar_{plugin_module}"
         ), f"Error extension missing fill_with_scalar_{plugin_module}"
+        assert hasattr(
```

Comment:
AHh, I see this is the answer to the above.Makes sense leaving it unimplemented in a few!